<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="editSearch"/>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonFind">
        <property name="text">
         <string>Искать</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="UploadButton">
        <property name="text">
         <string>Выгрузить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="TBrowser" name="textBrowser"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editID"/>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>модель</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editModel"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>год</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="editYear"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>цвет</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboColor">
        <item>
         <property name="text">
          <string>красный</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>зеленый </string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>синий</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>черный</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ButtonAdd">
        <property name="text">
         <string>добавить</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TBrowser</class>
   <extends>QTextBrowser</extends>
   <header location="global">tbrowser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ButtonFind</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>FindCar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>708</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>710</x>
     <y>28</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonAdd</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>AddCar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>755</x>
     <y>563</y>
    </hint>
    <hint type="destinationlabel">
     <x>725</x>
     <y>577</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UploadButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>UploadCar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>776</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>792</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonAdd</sender>
   <signal>clicked()</signal>
   <receiver>editID</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>734</x>
     <y>562</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>567</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonAdd</sender>
   <signal>clicked()</signal>
   <receiver>editModel</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>717</x>
     <y>552</y>
    </hint>
    <hint type="destinationlabel">
     <x>384</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ButtonAdd</sender>
   <signal>clicked()</signal>
   <receiver>editYear</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>783</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>566</x>
     <y>556</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>FindCar()</slot>
  <slot>AddCar()</slot>
  <slot>UploadCar()</slot>
 </slots>
</ui>
